### PR TITLE
Updating README with TinaCMS dev badge

### PR DESCRIPTION
**Pull request by PothieuG**
The goal of this PR is to integrate the Tina CMS Badge process into the README. The process will be as follows:
- Join the Discord server.
- Create a project with Tina.
- Record a video demonstrating completion.
- Validate the process by sending an email to badge@tina.io.

### CODE REVIEW

**Comment by bradystroud** [comment]
I wonder if there is a better term than "licence" (might get confused with the licence to the product)

**Comment by brookjeynes-ssw** [comment]
CC: @bradystroud @ncn-ssw 

Hey @PothieuG and @adamcogan ,
 
I have concerns about this idea.
 
Are these steps mandatory to complete before an open-source PR can be approved? I believe these steps will scare away more contributors than attract.
 
Step 1: I wouldn't make joining the Discord mandatory. Instead, I would mention this is where developer discussions takes place. This would help guide the contributor.
 
Step 2: It would be better to link off to a developer README.md instead, which would describe the process of how to test your changes locally on a TinaCMS project.
 
Step 3: Having to record a video unrelated to their issue and about their understanding of TinaCMS seems like a huge ask for someone and will result in this step either being skipped by most or cause a dev to not want to continue with the process further.
 
Steps 4-6: Making the contributor have to email/post the video and wait for their contributor license to be approved will lead to longer wait times and frustration. This will also consume SSW dev time.
 
These steps make contributing harder.

**Comment by wicksipedia** [comment]
Agree with @brookjeynes-ssw - as is, this may alienate the community as it can be perceived as a requirement to contribute back to the project.

> Hi, I am trying out TinaCMS with {{ TECH eg. NextJS }}...

This could be a suggestion when people join the Discord community as a template to follow on `#introductions`

Steps 2-6 - feel more internal SSW than anything else (especially the language/terms). I'd expect to see that on SugarLearning and not a public project. 

Also, I'm not sure that a 'done video' is enough to gauge whether someone is ready to use Tina on a client project.

For a cert, it may make more sense that a developer follows a tutorial that gets them to setup Tina, deploy it, then add some functionality. Then they submit the URL to that project, which can be checked (automated) for expected functionality

**Review by JackDevAU — COMMENTED** [review]
As per my conversation with @adamcogan @wicksipedia @bradystroud 

This needs some slight changes before going live. 

I'll make this a draft PR for the moment till they have been finalised.

File: README.md [hunk 1]
```diff
@@ -37,6 +37,9 @@ Visit [Tina's documentation](https://tina.io/docs/) to learn more.
 
 Visit the [GitHub Discussions](https://github.com/tinacms/tinacms/discussions) or our [Community Discord](https://discord.com/invite/zumN63Ybpf) to ask questions, or look us up on on Twitter at [@tinacms](https://twitter.com/tinacms).
 
+## 🧑‍🎓 Tina Creadly Badge
```
Comment (JackDevAU) [inline README.md:40]:
```suggestion
## 🧑‍🎓 Earn a Tina Badge
```

File: README.md [hunk 2]
```diff
@@ -37,6 +37,9 @@ Visit [Tina's documentation](https://tina.io/docs/) to learn more.
 
 Visit the [GitHub Discussions](https://github.com/tinacms/tinacms/discussions) or our [Community Discord](https://discord.com/invite/zumN63Ybpf) to ask questions, or look us up on on Twitter at [@tinacms](https://twitter.com/tinacms).
 
+## 🧑‍🎓 Tina Creadly Badge 
+
+If you want to go the extra mile and earn a Tina Dev Licence, follow this doc: [Earn a Tina Credly badge](./_docs/Dev-Licence.md)
```
Comment (JackDevAU) [inline README.md:42]:
```suggestion
[Earn a Tina Credly badge](./_docs/Dev-Licence.md)

Go the extra mile and earn a Tina Dev Licence 🦙
```

What about this? 😆

File: _docs/Dev-Licence.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+We encourage all developers to get their “TinaCMS licence” - that is, to get it up and running and complete the basics before implementing TinaCMS on a real project. However, this is a suggestion for those who wish to deepen their knowledge.  
+Here are the steps to complete your journey:
+
+1. Join [Tina's Discord channel](https://discord.com/invite/zumN63Ybpf) - You can leave a message on the `#introductions`, i.e.:
+
+    > Hi, I am trying out TinaCMS with {{ TECH eg. NextJS }}.  
+    > After I have evaluated TinaCMS, I plan to use it in the {{ INDUSTRY }} industry.  
+    > Previous CMSs I have used include {{ FILL IN }}.
+
+2. Create an Tina instance using the default template on [tina.io](https://tina.io) and [TinaCloud](https://app.tina.io/signin).
```
Comment (JackDevAU) [inline _docs/Dev-Licence.md:12]:
```suggestion
2. Create a Tina instance using the default template on [tina.io](https://tina.io) and [TinaCloud](https://app.tina.io/signin).
```

**Comment by wicksipedia** [comment]
@PothieuG 
1. PR needs a description
2. Is the email address setup? employee responsibility for monitoring it?

**Comment by PothieuG** [comment]
@wicksipedia 

> 1. PR needs a description

✅ Done, just added few seconds ago.

> 2. Is the email address setup? employee responsibility for monitoring it?

✅ Yes.

image
**Figure:** Warwick email mentioning the setup of badge@tina.io email being up and running

image
**Figure:** Tina badge validator responsabilities in the CRM's Employee responsabilities section

File: README.md [hunk 4]
```diff
@@ -37,6 +37,9 @@ Visit [Tina's documentation](https://tina.io/docs/) to learn more.
 
 Visit the [GitHub Discussions](https://github.com/tinacms/tinacms/discussions) or our [Community Discord](https://discord.com/invite/zumN63Ybpf) to ask questions, or look us up on on Twitter at [@tinacms](https://twitter.com/tinacms).
 
+## 🧑‍🎓 Earn a Tina Badge
+
+Go the extra mile and earn a Tina Dev Licence 🦙: [Earn a Tina Credly badge](./_docs/Dev-Licence.md)
```
Comment (wicksipedia) [inline README.md:42]:
```suggestion
Go the extra mile and earn a Tina Dev Licence 🦙: [Earn a Tina Credly badge](./_docs/dev-licence.md)
```

**Review by wicksipedia — DISMISSED** [review]

File: _docs/dev-licence.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+    > Hi, I am trying out TinaCMS with {{ TECH eg. NextJS }}.  
+    > After I have evaluated TinaCMS, I plan to use it in the {{ INDUSTRY }} industry.  
+    > Previous CMSs I have used include {{ FILL IN }}.
+
+2. Create a Tina instance using the default template on [tina.io](https://tina.io) and [TinaCloud](https://app.tina.io/signin).
+
+3. Record a [Done Video](https://www.ssw.com.au/rules/record-a-quick-and-dirty-done-video/) explaining how Tina works.
+
+    Here is a sample Done Video: https://www.youtube.com/watch?v=OhVYTOKCsWI
```
Comment (JackDevAU) [inline _docs/dev-licence.md:16]: Is this the video we want to use? Should we change this to a Tina related video? (Maybe someone doing this exact process?)

Comment (wicksipedia) [reply _docs/dev-licence.md]: dropped it, didn't make sense in the context of the video we want
added a list of things for us to look out for instead

File: _docs/dev-badge.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+## 🧑‍🎓 TinaCMS Dev Badge
+
+We encourage all developers to get their “TinaCMS badge” - that is, to get it up and running and complete the basics before implementing TinaCMS on a real project. This is a must to prove that you can use TinaCMS!
```
Comment (JackDevAU) [inline _docs/dev-badge.md:3]:
```suggestion
We encourage all developers to earn their “TinaCMS badge” by getting TinaCMS up and running and complete the basics before implementing TinaCMS on a real project. This is essential to prove that you can effectively use TinaCMS!
```

File: _docs/dev-badge.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+     Previous CMSs I have used include {{ FILL IN }}
+
+2. Get TinaCMS running and deploy a site using [tina.io](https://tina.io) and [TinaCloud](https://app.tina.io/signin)
+   
+   Feel free to start with one of the starter templates
+   This can be a personal site, or one you did for work
+
+3. Record a [Done Video](https://www.ssw.com.au/rules/record-a-quick-and-dirty-done-video/) explaining how Tina works
+
+   What we are looking for:
```
Comment (JackDevAU) [inline _docs/dev-badge.md:20]:
```suggestion
   Here's what we're looking for:
```

File: _docs/dev-badge.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+1. Join [Tina's Discord channel](https://discord.com/invite/zumN63Ybpf) - You can leave a message on the `#introductions`, i.e.:
+
+   > Hi, I am trying out TinaCMS with {{ TECH eg. NextJS }}
+     After I have evaluated TinaCMS, I plan to use it in the {{ INDUSTRY }} industry.  
+     Previous CMSs I have used include {{ FILL IN }}
+
+2. Get TinaCMS running and deploy a site using [tina.io](https://tina.io) and [TinaCloud](https://app.tina.io/signin)
+   
+   Feel free to start with one of the starter templates
+   This can be a personal site, or one you did for work
```
Comment (JackDevAU) [inline _docs/dev-badge.md:16]:
```suggestion
   This can be a personal site or one you’ve built for work
```

File: _docs/dev-badge.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+3. Record a [Done Video](https://www.ssw.com.au/rules/record-a-quick-and-dirty-done-video/) explaining how Tina works
+
+   What we are looking for:
+   - Who you are
+   - Why you are using TinaCMS
+   - Something you like about TinaCMS
+   - Any feedback or suggestions you have from your initial tryout
+     Tip: Have each suggestion's discussion or issue open in a separate tab
+
+4. Post the vidoe to YouTube (or other video platform) - make sure that it is public
```
Comment (JackDevAU) [inline _docs/dev-badge.md:27]:
```suggestion
4. Post the video to YouTube (or another video platform) - make sure that it is public
```

File: _docs/dev-badge.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+5. (Optional) Post your video on social media (e.g., LinkedIn, Discord, X...)
+
+6. Let us know about it! Email badge@tina.io
```
Comment (JackDevAU) [inline _docs/dev-badge.md:30]:
```suggestion
6. Let us know about it! Email us at badge@tina.io
```

File: _docs/dev-badge.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+   Include a link to:
+   - your video
+   - links to your posts
```
Comment (JackDevAU) [inline _docs/dev-badge.md:34]:
```suggestion
   - Your social media posts (if applicable)
```

File: _docs/dev-badge.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+   - Any feedback or suggestions you have from your initial tryout
+     Tip: Have each suggestion's discussion or issue open in a separate tab
+
+4. Post the vidoe to YouTube (or other video platform) - make sure that it is public
+5. (Optional) Post your video on social media (e.g., LinkedIn, Discord, X...)
+
+6. Let us know about it! Email badge@tina.io
+
+   Include a link to:
+   - your video
```
Comment (JackDevAU) [inline _docs/dev-badge.md:33]:
```suggestion
   - Your video
```

File: _docs/dev-badge.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+   - Something you like about TinaCMS
+   - Any feedback or suggestions you have from your initial tryout
+     Tip: Have each suggestion's discussion or issue open in a separate tab
+
+4. Post the vidoe to YouTube (or other video platform) - make sure that it is public
+5. (Optional) Post your video on social media (e.g., LinkedIn, Discord, X...)
+
+6. Let us know about it! Email badge@tina.io
+
+   Include a link to:
```
Comment (JackDevAU) [inline _docs/dev-badge.md:32]:
```suggestion
   Include links to:
```

**Review by JackDevAU — APPROVED** [review]
🦙

**Pull request merged**